source/css/post.css

```diff
@@ -22,6 +22,26 @@
     color: color-mod(var(--accent) a(70%));
   }
 
+  &-lastmod {
+    --border: 3px dotted var(--accent);
```

Comment:
I'm not sure about the heavy styling for this kind of element. Could you show me a screenshot of this?

Reply:
This is the rendering of the desktop:
![desktop](https://user-images.githubusercontent.com/32632779/68303664-4abf9d00-00df-11ea-803a-6835887e14f9.png)

And the mobile:
![mobile](https://user-images.githubusercontent.com/32632779/68303665-4abf9d00-00df-11ea-9be0-012dfc9bb1ef.png)

We will find that if the update time and the creation time are on the same line, a line break will occur when the website is accessed via mobile phone. I don't know how to improve the design, so I simply put the update time at the end of the post. If there is a more appropriate treatment, please let me know, thank you : D

Reply:
Very sorry for a late response! I totally forgot about some of the PRs 😞 

About the styling. What do you think about doing something like this instead:
`2020-02-21 [Modified: 2020-03-03] :: Radek`

We can use config file to set any string we want to replace `Modified` or `Updated` (it's up to you what it should be by the default).

I'd like to keep this as minimal as possible 🙏

Reply:
Never mind, I actually forgot about this PR 😉。I think your idea is much better than mine, I will make some changes later and resubmit.

Reply:
I made some changes and now it looks like this:
![demo](https://user-images.githubusercontent.com/32632779/76432894-8c928400-63ee-11ea-9618-7bee4445f9a1.png)

And I added a nullable option named `updatedDatePrefix` (the default is Updated).Do you think it needs to be modified? :D